troposphere/ec2.py

```diff
@@ -757,13 +757,47 @@ class LaunchSpecifications(AWSProperty):
     }
 
 
+class SpotFleetLaunchTemplateSpecification(AWSProperty):
+    props = {
+        'LaunchTemplateId': (basestring, False),
+        'LaunchTemplateName': (basestring, False),
+        'Version': (basestring, True),
+    }
+
+    def validate(self):
+        template_identifiers = [
+            'LaunchTemplateId',
+            'LaunchTemplateName'
+        ]
+        exactly_one(self.__class__.__name__, self.properties, template_identifiers)
+
+
+class LaunchTemplateOverrides(AWSProperty):
+    props = {
+        'AvailabilityZone': (basestring, False),
+        'InstanceType': (basestring, False),
+        'SpotPrice': (basestring, False),
+        'SubnetId': (basestring, False),
+        'WeightedCapacity': (basestring, False)
+    }
+
+
+class LaunchTemplateConfig(AWSProperty):
+    props = {
+        'LaunchTemplateSpecification': (SpotFleetLaunchTemplateSpecification, True),
+        'Overrides': ([LaunchTemplateOverrides], False)
+    }
+
+
 class SpotFleetRequestConfigData(AWSProperty):
     props = {
         'AllocationStrategy': (basestring, False),
         'ExcessCapacityTerminationPolicy': (basestring, False),
         'IamFleetRole': (basestring, True),
+        'InstanceInterruptionBehavior': (basestring, False),
         'ReplaceUnhealthyInstances': (boolean, False),
-        'LaunchSpecifications': ([LaunchSpecifications], True),
+        'LaunchSpecifications': ([LaunchSpecifications], False),
```

Comment:
Docs still say LaunchSpecifications is required -- but it's not required when LaunchTemplateConfigs is supplied. PR submitted here: https://github.com/awsdocs/aws-cloudformation-user-guide/pull/161